.github/workflows/testing.yml

```diff
@@ -11,19 +11,145 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: |
-          npm install
-      - run: |
-          npm run build
-      - run: |
-          npm run package
-      - run: |
-          npm run test
-
-  test-action: # make sure the action works on a clean machine without building
-      runs-on: ubuntu-latest
-      steps:
-        - uses: actions/checkout@v2
-        - uses: ./
-        - name: Check that result file is generated
-          run: tree -L 2 .tortellini
+      - run: npm install
+      - run: npm run build
+      - run: npm run package
+      - run: npm run test
+
+
+  no-args:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Clone this repository
+          uses: actions/checkout@v2
+        - name: Test without any input arguments
+          uses: ./
+        - name: Check the resulting directory tree
+          run: tree -a -L 5 -I '.git' .
+        - name: Verify that we have the analyze result
+          run: ls -l out/analyzer-result.yml
```

Comment:
Can you also run `ls` to check
analyzer-result.yml
evaluation-result.yml
gl-license-scanning-report.json

Reply:
also use `.tortellini/out/analyzer-result.yml`